camviewer_ui_impl.py

```diff
@@ -142,6 +142,8 @@ def __init__(self, app, main):
         self.last = QPoint(0, 0)
 
     def eventFilter(self, obj, event):
+        if event.type() in [QEvent.MouseButtonPress, QEvent.KeyPress]:
+            self.main.refresh_disco()
```

Comment:
🪩 ?

does this stand for disconnect? (I understand what it does from further reading, but the name struck me)

Reply:
Is it worth fixing all of these annoying names? It wouldn't be so hard.

Reply:
I will fix the annoying names that are related to this PR:
- The ones that are too terse
- The ones that use camelCase

Reply:
Also:
- The ones whos names make no sense now that I'm not disconnecting any cameras!

I think I was too tepid with renames on the first pass because I was afraid of breaking random undocumented internals.

Reply:
I think future us would appreciate it, in the whole "leave it better than you found it" frame of mind

Reply:
No more panic at the disco, I renamed nearly everything related to this PR to be much closer to what it actually does